### ISSUE TITLE
Ability to make ContextMenu floating

### ISSUE THREAD

**Issue by notandy** [sapcc/juno#347]
It would be nice to have the pop-upped `Menu` of the `ContextMenu` floated/fixed instead of taking space inside the element the 3-dots button is located.

**Comment by edda** [comment]
Agreed! The `ContextMenu` was created before we had a way to position floated elements properly. Thanks for reminding us!
